driver/bpf/filler_helpers.h

```diff
@@ -19,6 +19,11 @@ or GPL2.txt for full copies of the license.
 
 #include "../ppm_flag_helpers.h"
 
+static inline bool in_port_range(uint16_t port, uint16_t min, uint16_t max)
```

Comment:
Please consider using `__always_inline` instead of `inline` to be consistent with the rest of the code: the reason is, in bpf you cannot really have a non-inline function, since the whole program is a single function. However, LLVM does not know this and will happily choose to not respect the `inline` keyword occasionally (it happened during development) and put a real function call. Debugging that is kind of a nightmare, because it will just mysteriously fail when the BPF loader tries to relocate the code at runtime, with no hint as to what is the issue, so it's just better to prevent it.

Reply:
Good to know, thanks! Fixed.